docs/documentation.html

```diff
@@ -69,15 +69,15 @@
     <h2><a id="connect" href="#connect">8. Kafka Connect</a></h2>
     <!--#include virtual="connect.html" -->
 
-    <h2><a id="streams" href="/0102/documentation/streams">9. Kafka Streams</a></h2>
+    <h2><a id="streams" href="/0110/documentation/streams">9. Kafka Streams</a></h2>
     <p>
-        Kafka Streams is a client library for processing and analyzing data stored in Kafka and either writes the resulting data back to Kafka or sends the final output to an external system. It builds upon important stream processing concepts such as properly distinguishing between event time and processing time, windowing support, exactly-once processing semantics and simple yet efficient management of application state.
+        Kafka Streams is a client library for processing and analyzing data stored in Kafka. It builds upon important stream processing concepts such as properly distinguishing between event time and processing time, windowing support, exactly-once processing semantics and simple yet efficient management of application state.
     </p>
     <p>
         Kafka Streams has a <b>low barrier to entry</b>: You can quickly write and run a small-scale proof-of-concept on a single machine; and you only need to run additional instances of your application on multiple machines to scale up to high-volume production workloads. Kafka Streams transparently handles the load balancing of multiple instances of the same application by leveraging Kafka's parallelism model.
     </p>
 
-    <p>Learn More about Kafka Streams read <a href="/0102/documentation/streams">this</a> Section.</p>
+    <p>Learn More about Kafka Streams read <a href="/0110/documentation/streams">this</a> Section.</p>
```

Comment:
@derrickdoo and I have been discussing about this, and the conclusion is that in the entry page `documentation.html` it is much trickier to do and hence we did not do that.